app/client/ui/DescriptionConfig.ts

```diff
@@ -11,8 +11,9 @@ const t = makeT('FieldConfig');
 
 export function buildDescriptionConfig(
     owner: MultiHolder,
-    origColumn: ColumnRec,
+    parent: ColumnRec | ViewSectionRec,
     cursor: ko.Computed<CursorPos>,
+    testPrefix: string,
```

Comment:
Sorry for nitpicking, but can you wrap these two parameters into a single `options` parameter?

```suggestion
    options: {
      cursor: ko.Computed<CursorPos>,
      testPrefix: string,
    }    
```